config/v1/types_feature.go

```diff
@@ -112,6 +112,16 @@ type FeatureGateStatus struct {
 	// +listType=map
 	// +listMapKey=version
 	FeatureGates []FeatureGateDetails `json:"featureGates"`
+
+	// renderedMinimumComponentVersions are the component versions that the feature gate list of this status were rendered from.
+	// Currently, the only supported component is Kubelet, and setting a required minimum kubelet component will set the
+	// minimumKubeletVersion field in the nodes.config.openshift.io CRD.
+	// +kubebuilder:validation:MaxItems:=1
+	// +listType=map
+	// +listMapKey=component
+	// +openshift:enable:FeatureGate=MinimumKubeletVersion
+	// +optional
+	RenderedMinimumComponentVersions []RequiredMinimumComponentVersion `json:"renderedMinimumComponentVersions,omitempty"`
```

Comment:
I think this wants to be in `FeatureGateDetails` doesn't it? FeatureGateDetails shows a list of feature gates states, at a particular version, and shows the different versions over time.

We aren't going to re-render the old items in the list when the component version changes, which makes me think this is a property of a version, and not a property of the entire feature gate

What is going to read from this? The feature gate accessor currently finds the current matching version entry from the list, so I suspect it is correct to add it there

Reply:
good point, moved